.gitignore

```diff
@@ -65,3 +65,42 @@ agenta-backend/agenta_backend/migrations/migrations/v0_11_0_to_v0_12_0/202402011
 agenta-backend/agenta_backend/migrations/migrations/v0_11_0_to_v0_12_0/20240203035520_add_workspace_field_to_db_models.py
 docker-compose.oss.prod.yml
 agenta-backend/agenta_backend/services/benchmark.ipynb
+examples/baby_name_generator/docker/_app.py
```

Comment:
Maybe find a more generic way to organize and ignore these files ? examples below
- `examples/**/docker/*`
- `examples/**/*.pyc`
- `examples/**/tests/*` (require `examples/baby_name_generator/test_example/` to become `examples/baby_name_generator/tests/example/`)
- ...